<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pilotee</class>
 <widget class="QDialog" name="pilotee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/img/bg3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> *{
font-family:century gothic;
color:#1990ea;
border-color:#1990ea;
font-size:70px;}</string>
   </property>
   <property name="text">
    <string>Welcome Ahmed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1322">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>750</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: #1990ea;
color: white;
font-size:20px;
border-radius:15px;}  
QPushButton:hover
{
    border: 1px solid #1990ea;
color:  #1990ea;
border-radius:15px;
background: white;}  </string>
   </property>
   <property name="text">
    <string>Supprimer compte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_disc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: #1990ea;
color: white;
font-size:20px;
border-radius:15px;}  
QPushButton:hover
{
    border: 1px solid #1990ea;
color:  #1990ea;
border-radius:15px;
background: white;}  </string>
   </property>
   <property name="text">
    <string>Disconnecter</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_historique">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>821</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> *{
font-family:century gothic;
color:#1990ea;
border-color:#1990ea;
font-size:14px;}
	#frame_2{


font-family:century gothic;
    border: 1px solid #1990ea;
    border-radius: 30px;
border-color: lightgray;
    margin-top: 0.5em;
background : white;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font: 40pt  'Oswald';font-size:30px;
color: #1990ea;
}</string>
    </property>
    <property name="text">
     <string>Historique</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>761</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">



QScrollBar:vertical {
    background: rgb(188, 224, 235);
            border: 0px solid #999999;
            width:10px;    

margin: 0px 0px 0px 0px;
}
 QScrollBar::handle:vertical {
    background: rgb(71, 153, 176);
          	border: 0px solid red;
            min-height: 0px;
			border-radius: 8px;

	 }
QScrollBar:horizontal {
    background: rgb(188, 224, 235);
            border: 0px solid #999999;
margin: 0px 0px 0px 0px;

}
 QScrollBar::handle:horizontal {
    background: rgb(71, 153, 176);
                     	border: 0px solid red;
            min-height: 0px;
			border-radius: 8px;
            width:10px;    


 }</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ContiguousSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_pdfhistorique">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #1990ea;
color: white;
border-radius:15px;}  
QPushButton:hover
{
    border: 1px solid #1990ea;
color:  #1990ea;
border-radius:15px;
background: white;}  </string>
    </property>
    <property name="text">
     <string>Imprimer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/img/img/sort.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>tableView_4</zorder>
   <zorder>label_22</zorder>
   <zorder>pushButton_pdfhistorique</zorder>
   <zorder>label_24</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>710</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
